CharlieBackend.Panel/Services/StudentGroupService.cs

```diff
@@ -43,8 +46,14 @@ public async Task<IList<StudentGroupViewModel>> GetAllStudentGroupsAsync()
             var studentGroupsResponse = await _apiUtil.GetAsync<IList<StudentGroupDto>>(getAllStudentGroupsEndpoint);
             var studentGroups = _mapper.Map<IList<StudentGroupViewModel>>(studentGroupsResponse);
             
-            var students = await _studentService.GetAllStudentsAsync(); ;
-            var mentors = await _mentorService.GetAllMentorsAsync();
+            var students = await _studentService.GetAllStudentsAsync();
+
+            var mentors = await _mentorService.GetAllActiveMentorsAsync();
+            if (!(_currentUserService.Role == Core.Entities.UserRole.Mentor))
```

Comment:
Not a very clear condition, in my opinion it's more understandable
if (_currentUserService.Role != Core.Entities.UserRole.Mentor)